plugwise/util.py

```diff
@@ -115,7 +115,7 @@ def check_model(name: str | None, vendor_name: str | None) -> str | None:
     if name is not None and "lumi.plug" in name:
         return "Aqara Smart Plug"
 
-    return name
+    return name  # pragma: no cover
```

Comment:
We should check on this one, since it's the default it would be weird as being untestable?

Reply:
This is the fallback-case, I could leave it out but then ruff will complain: missing return.

Reply:
Let's leave it be :)

Reply:
This return can be removed apparently.

Reply:
No it can't, mypy now complains :(